beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/BlockFactoryPhase0.java

```diff
@@ -66,6 +72,34 @@ public SafeFuture<BlockContainer> createUnsignedBlock(
         .thenApply(BeaconBlockAndState::getBlock);
   }
 
+  @Override
+  public SafeFuture<BlockContainer> createUnsignedBlock(
+      final BeaconState blockSlotState,
+      final UInt64 newSlot,
+      final BLSSignature randaoReveal,
+      final Optional<Bytes32> optionalGraffiti) {
+    checkArgument(
+        blockSlotState.getSlot().equals(newSlot),
+        "Block slot state for slot %s but should be for slot %s",
+        blockSlotState.getSlot(),
+        newSlot);
+
+    // Process empty slots up to the one before the new block slot
+    final UInt64 slotBeforeBlock = newSlot.minus(UInt64.ONE);
+
+    final Bytes32 parentRoot = spec.getBlockRootAtSlot(blockSlotState, slotBeforeBlock);
+
+    return spec.createNewUnsignedBlock(
+            newSlot,
+            spec.getBeaconProposerIndex(blockSlotState, newSlot),
+            blockSlotState,
+            parentRoot,
+            operationSelector.createSelector(
+                parentRoot, blockSlotState, randaoReveal, optionalGraffiti),
+            produceBlindedBlocks)
```

Comment:
AS per discussion, I think we should avoid having this flag attached to the existing validator config.
I can see it hardcoded to `true` or driven by another beacon-node-specific `--X` parameter

Reply:
Done, removed the validator config option